src/MOI_wrapper/MOI_nonlinear.jl

```diff
@@ -194,6 +196,7 @@ function MOI.add_constraint(
     model::Optimizer,
     f::MOI.ScalarNonlinearFunction,
     s::_SCALAR_SETS,
+    resvar_sense::_SCALAR_SETS = s,
```

Comment:
You cannot define this method.

Reply:
Thanks! Yeah I'm not sure what I was thinking here 😅